sympy/printing/smtlib.py

```diff
@@ -0,0 +1,526 @@
+import builtins
+import typing
+
+import sympy
+from sympy.core import Add, Mul
+from sympy.core import Symbol, Expr, Float, Rational, Integer, Basic
+from sympy.core.function import UndefinedFunction, Function
+from sympy.core.relational import Relational, Unequality, Equality, LessThan, GreaterThan, StrictLessThan, StrictGreaterThan
+from sympy.functions.elementary.complexes import Abs
+from sympy.functions.elementary.exponential import exp, log, Pow
+from sympy.functions.elementary.hyperbolic import sinh, cosh, tanh
+from sympy.functions.elementary.miscellaneous import Min, Max
+from sympy.functions.elementary.piecewise import Piecewise
+from sympy.functions.elementary.trigonometric import sin, cos, tan, asin, acos, atan, atan2
+from sympy.logic.boolalg import And, Or, Xor, Implies, Boolean
+from sympy.logic.boolalg import BooleanTrue, BooleanFalse, BooleanFunction, Not, ITE
+from sympy.printing.printer import Printer
+from sympy.sets import Interval
+
+
+class SMTLibPrinter(Printer):
+    printmethod = "_smt"
```

Comment:
Why `_smt` and not `_smtlib`?

Reply:
No reason, just felt more natural. Will fix.